tests/tck/features/match/MultiLineMultiQueryParts.feature

```diff
@@ -0,0 +1,272 @@
+# Copyright (c) 2021 vesoft inc. All rights reserved.
+#
+# This source code is licensed under Apache 2.0 License.
+Feature: Multi Line Multi Query Parts
+
+  Background:
+    Given a graph with space named "nba"
+
+  Scenario: Multi Line Multi Path Patterns
+    When executing query:
+      """
+      USE nba;
+      MATCH (m)-[]-(n), (n)-[]-(l) WHERE id(m)=="Tim Duncan"
+      RETURN m.player.name AS n1, n.player.name AS n2,
+      CASE WHEN l.team.name is not null THEN l.team.name
+      WHEN l.player.name is not null THEN l.player.name ELSE "null" END AS n3 ORDER BY n1, n2, n3 LIMIT 10
+      """
+    Then the result should be, in order:
+      | n1           | n2            | n3           |
+      | "Tim Duncan" | "Aron Baynes" | "Celtics"    |
+      | "Tim Duncan" | "Aron Baynes" | "Pistons"    |
+      | "Tim Duncan" | "Aron Baynes" | "Spurs"      |
+      | "Tim Duncan" | "Aron Baynes" | "Tim Duncan" |
+      | "Tim Duncan" | "Boris Diaw"  | "Hawks"      |
+      | "Tim Duncan" | "Boris Diaw"  | "Hornets"    |
+      | "Tim Duncan" | "Boris Diaw"  | "Jazz"       |
+      | "Tim Duncan" | "Boris Diaw"  | "Spurs"      |
+      | "Tim Duncan" | "Boris Diaw"  | "Suns"       |
+      | "Tim Duncan" | "Boris Diaw"  | "Tim Duncan" |
+    When executing query:
+      """
+      USE nba;
+      MATCH (m)-[]-(n), (l)-[]-(n) WHERE id(m)=="Tim Duncan"
+      RETURN m.player.name AS n1, n.player.name AS n2,
+      CASE WHEN l.team.name is not null THEN l.team.name
+      WHEN l.player.name is not null THEN l.player.name ELSE "null" END AS n3 ORDER BY n1, n2, n3 LIMIT 10
+      """
+    Then the result should be, in order:
+      | n1           | n2            | n3           |
+      | "Tim Duncan" | "Aron Baynes" | "Celtics"    |
+      | "Tim Duncan" | "Aron Baynes" | "Pistons"    |
+      | "Tim Duncan" | "Aron Baynes" | "Spurs"      |
+      | "Tim Duncan" | "Aron Baynes" | "Tim Duncan" |
+      | "Tim Duncan" | "Boris Diaw"  | "Hawks"      |
+      | "Tim Duncan" | "Boris Diaw"  | "Hornets"    |
+      | "Tim Duncan" | "Boris Diaw"  | "Jazz"       |
+      | "Tim Duncan" | "Boris Diaw"  | "Spurs"      |
+      | "Tim Duncan" | "Boris Diaw"  | "Suns"       |
+      | "Tim Duncan" | "Boris Diaw"  | "Tim Duncan" |
```

Comment:
What's the reason to have these 2 cases? Aren't they both in form of  `use...; match...` ?

Reply:
confirm that some case.

Reply:
For me, these 2 cases are describing the same scenario. Consider removing it later.